docs/setup/install-with-keink.md

```diff
@@ -0,0 +1,121 @@
+# Deploying KubeEdge with Keink
```

Comment:
add a head with title and sidebar_position ref to https://github.com/kubeedge/website/blob/master/docs/setup/install-with-binary.md?plain=1  

The sidebar_position determines the order of the file in the directory. I think it's better to put it after install-with-binary. How about you?  Accordingly, the sidebar_position number of the other files should be adjusted.  
![image](https://github.com/kubeedge/website/assets/48788150/7be95f36-97a7-4d29-9ff2-2ce0daf7802b)